shell/src/main/java/alluxio/cli/job/command/ListCommand.java

```diff
@@ -58,13 +57,14 @@ public void validateArgs(CommandLine cl) throws InvalidArgumentException {
   }
 
   @Override
-  public int run(CommandLine cl) throws AlluxioException, IOException {
+  public int run(CommandLine cl) {
     try (CloseableResource<JobMasterClient> client = JobContext
         .create(mFsContext.getClusterConf(), mFsContext.getClientContext().getUserState())
         .acquireMasterClientResource()) {
-      List<Long> ids = client.get().list();
-      for (long id : ids) {
-        System.out.println(id);
+      List<JobInfo> jobInfos = client.get().listDetailed();
+      for (JobInfo jobInfo : jobInfos) {
+        System.out.println(String.format("%-15s %-10s %-10s", jobInfo.getId(), jobInfo.getName(),
```

Comment:
Maybe tab delimit for prettiness?

Reply:
Do you prefer that over this? I figured aligning this way is a lot cleaner.

Reply:
(There are more white spaces in between then the example shown because github truncates extra white spaces)